lib/galaxy/webapps/galaxy/api/library_contents.py

```diff
@@ -101,19 +98,7 @@ def traverse(folder):
                     rval.append(ld)
             return rval
 
-        decoded_library_id = self.decode_id(library_id)
-        try:
-            library = (
-                trans.sa_session.query(trans.app.model.Library)
-                .filter(trans.app.model.Library.table.c.id == decoded_library_id)
-                .one()
-            )
-        except MultipleResultsFound:
-            raise exceptions.InconsistentDatabase("Multiple libraries found with the same id.")
-        except NoResultFound:
-            raise exceptions.RequestParameterInvalidException("No library found with the id provided.")
```

Comment:
> The first one is not possible:

That's why I commented on the line I commented on, instead of a range of lines.


> We retrieve records by id everywhere and rarely check for this error.

We sure check this in many, if not all places where values come in from the API. Every missing check for incoming values is a bug, no need to make it worse here.

Reply:
Done. Sorry - I was responding from PR's main page - 4 lines were displayed as highlighted, I didn't realize you commented on the last one only.